packages/x-tree-view-pro/src/internals/plugins/useTreeViewItemsReordering/useTreeViewItemsReordering.itemPlugin.ts

```diff
@@ -90,7 +90,13 @@ export const useTreeViewItemsReorderingItemPlugin: TreeViewItemPlugin = ({ props
             return;
           }
 
-          instance.stopDraggingItem(itemId);
+          // Check if the drag-and-drop was cancelled, possibly by pressing Escape
+          if (event.dataTransfer.dropEffect === 'none') {
+            instance.cancelDragging();
```

Comment:
```suggestion
            instance.cancelDraggingItem();
```

Nitpick, to be consistent with `instance.completeDraggingItem`